app/controllers/course/assessment/categories_controller.rb

```diff
@@ -0,0 +1,36 @@
+class Course::Assessment::CategoriesController < Course::Assessment::Controller
+  load_and_authorize_resource :category,
+                              through: :course,
+                              through_association: :assessment_categories,
+                              class: Course::Assessment::Category.name
+  add_breadcrumb :admin, :course_admin_path
```

Comment:
then this controller won't need to manually deal with every part of the breadcrumb.